apps/portals/src/configurations/elportal/synapseConfigs/cohortbuilder.ts

```diff
@@ -10,8 +10,8 @@ import { handleSelectedFilesToParticipants } from './handleFilesToParticipants'
 import {
   QueryWrapperSynapsePlotProps,
   QueryWrapperSynapsePlotRowClickEvent,
-} from 'synapse-react-client/src/components/QueryWrapperPlotNav/QueryWrapperSynapsePlot'
```

Comment:
/src/components/... is not exported by synapse-react-client, so rewrite this import (synapse-react-client was also modified to export these types)